capitalone/rubicon-ml
# docs cleanup

closes: #102 

---

I noticed a handful of little things I wanted to address in the docs while I was fixing the whitesource stuff and I really wanted to deal with the hour long docs builds

## What
  * only installs the `rubicon_ml` dependencies that are required to build the docs
    * this involves manually installing a local `rubicon_ml` via `pip` with the `--no-deps` flag, which can't be done in an environment yaml
    * updated docs, readmes and workflows to reflect this change
  * removes `build-docs.sh` in favor of just the Makefile
    * removed/reorganized some things the Makefile didn't need to do anymore
  * adds the `sklearn` submodule to the library reference
  * cleans up unused/outdated docs assets
    * there were extra images and gifs, like all the old lab extension stuff 
    * reorganized the custom css file
  * adds a link to the header of each notebook in the docs to run it on Binder

## How to Test
  * the docs are live - they built in 2 minutes!
    * make sure the library reference looks like it has everything
    * check the github and binder links on the docs notebooks
    * validate that readmes and contributing docs reflect these changes
